src/channels.jl

```diff
@@ -3,50 +3,75 @@
 ##
 
 function displace(::Type{Td}, ::Type{Tt}, basis::SymplecticBasis{N}, alpha::A, noise::M) where {Td,Tt,N<:Int,A,M}
+    disp_type, transform_type = _infer_types(Td, Tt, basis)
```

Comment:
Really minor and stupid request, but could we change it to this
```suggestion
    dtype, ttype = _infer_types(Td, Tt, basis)
```
I'm a minimalist and don't like too much text in a code name :)

Reply:
Thank you!